clients/java/client/src/main/java/org/camunda/bpm/client/task/impl/dto/SetVariablesRequestDto.java

```diff
@@ -24,14 +24,14 @@
 
 public class SetVariablesRequestDto extends RequestDto {
```

Comment:
🙃  You could consider renaming the class as well for consistency.

Reply:
I was thinking about renaming it to `Modify..` or something but with this API users can also set new variables and update existing ones. So I think `Set..` is not a bad name after all.